<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_promo_event</class>
 <widget class="QMainWindow" name="gestion_promo_event">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>951</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/res/img/hand-painted-watercolor-background-with-sky-clouds-shape_24972-1095.jpg);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Promotion</string>
     </attribute>
     <widget class="QLineEdit" name="rech_promo">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="table_promo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>71</y>
        <width>651</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#5500ff;&quot;&gt;Gestion Promotions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="promo_sel_col">
      <property name="geometry">
       <rect>
        <x>591</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Produits</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nouv Prix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Supermarche</string>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>50</y>
        <width>221</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Form</string>
      </property>
      <widget class="QComboBox" name="supermarche">
       <property name="geometry">
        <rect>
         <x>91</x>
         <y>210</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Produits</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Supermarche</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actionedit_promo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actionadd_promo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actiondelete_promo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="prix">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>60</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="produit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="nouv_prix">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>180</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nouv Prix</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="export_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>651</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Excel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Magasins</string>
     </attribute>
     <widget class="QTableView" name="table_evenement">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>651</width>
        <height>341</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#5500ff;&quot;&gt;Gestion magasins&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech_evenement">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="evenement_sel_col">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Produits</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Debut</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Categorie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Promo</string>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>40</y>
        <width>211</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Form</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Promo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_2">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="promot">
       <property name="geometry">
        <rect>
         <x>91</x>
         <y>260</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QComboBox" name="categorie">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>cat 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cat 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cat 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cat 4</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Categorie</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actionadd_evenement">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actionedit_evenement">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="actiondelete_evenement">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>94</x>
         <y>60</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QDateEdit" name="debut">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDateEdit" name="fin">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Debut</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Fin</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="produit_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>150</y>
         <width>104</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Produit</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>E-Mail</string>
     </attribute>
     <widget class="QTextEdit" name="msg">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>126</y>
        <width>871</width>
        <height>291</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>type your message here ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mail_pass">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>your mail password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>701</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>attachement here</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>871</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseBtn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>90</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse ...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rcpt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>611</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>recipient</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subject">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>54</y>
        <width>871</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>subject</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
